product_docs/docs/pem/8/pem_query_tool.mdx

```diff
@@ -238,26 +238,26 @@ Use the `Connection status` feature to view the current connection and transacti
 
 ## Macros
 
-Query Tool Macros enable you to execute pre-defined SQL queries with a single key press. Pre-defined queries can contain the placeholder $SELECTION$. Upon macro execution, the placeholder will be replaced with the currently selected text in the Query Editor pane of the Query Tool.
+Query Tool Macros enable you to execute pre-defined SQL queries with a single key-press. Pre-defined queries can contain the placeholder $SELECTION$. Upon macro execution, the placeholder will be replaced with the currently selected text in the Query Editor pane of the Query Tool.
 
 ![Query Tool - Connection Status button](images/query_tool_manage_macros.png)
 
-To create a macro, select the *Manage Macros* option from the *Macros* menu on the *Query Tool*. Select the key you wish to use, enter the name of the macro, and the query, optionally including the selection placeholder, and then click the *Save* button to store the macro.
+To create a macro, select the `Manage Macros` option from the Macros menu on the Query Tool. Select the key you wish to use, enter the name of the macro, and the query, optionally including the selection placeholder, and then click the `Save` button to store the macro.
 
 ![Query Tool - Connection Status button](images/query_tool_manage_macros_dialog.png)
 
-To clear a macro, select the macro on the *Manage Macros* dialogue, and then click the *Clear* button.
+To clear a macro, select the macro on the `Manage Macros` dialogue, and then click the `Clear` button.
```

Comment:
No tagging for dialog box names. Other verbiage issues we've discussed before. Put the name of the control where you're working first:

To clear a macro, in the Manage Macros dialog box, select the macro and  select **Clear**.